gost94/src/params.rs

```diff
@@ -31,6 +34,14 @@ impl Gost94Params for CryptoProParam {
     const NAME: &'static str = "Gost94CryptoPro";
 }
 
+#[cfg(feature = "oid")]
+impl AssociatedOid for CryptoProParam {
+    /// Per the RFC 4490, this OID is used for the GOST R 34.11-94 hash with CryptoPro params.
+    /// The OID 1.2.643.2.2.30.1 is used in the PublicKey params to denote CryptoPro paramset, but
+    /// not the hash function itself.
```

Comment:
According to this description we should use:
```rust
impl AssociatedOid for Gost94Core<CryptoProParam> {
    const OID: ObjectIdentifier = ObjectIdentifier::new_unwrap("1.2.643.2.2.9");
}

impl AssociatedOid for CryptoProParam {
    const OID: ObjectIdentifier = ObjectIdentifier::new_unwrap("1.2.643.2.2.30.1");
}
```

Also comment like this probably should go the crate docs, otherwise it will be hard for users to discover it.

Reply:
Done